### DISCUSSION TITLE
Tic Tac Toe

### DISCUSSION THREAD

**Discussion by NicoMartina** [reactplay/react-play#716] [Ideas]
Hello, I am starting a Tic Tac Tor project and I am new to React. Havent ever done a project on my own. First time using react-play. So I am not sure how to begin, any help is useful! 🙏🙏

**Comment by atapas** [comment]
> Hello, I am starting a Tic Tac Tor project and I am new to React. Havent ever done a project on my own. First time using react-play. So I am not sure how to begin, any help is useful! 🙏🙏

Just get started. I would suggest you join our Discord where we discuss, learn, and build.

**Reply by NicoMartina** [reply]
Which is the discord? Did not see it

**Reply by atapas** [reply]
I see you are in already! :)
